kodilla-rps/src/main/java/com/kodilla/rps/Game.java

```diff
@@ -0,0 +1,168 @@
+            return true;
+        } else if (computerMove == LIZARD && (playerMove == PAPER || playerMove == SPOCK)) {
+            return true;
+        } else if (computerMove == SPOCK && (playerMove == SCISSORS || playerMove == ROCK)) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    private boolean draw(Moves playerMove, Moves computerMove) {
+        if (playerMove == ROCK && computerMove == ROCK) {
+            return true;
+        } else if (playerMove == PAPER && computerMove == PAPER) {
+            return true;
+        } else if (playerMove == SCISSORS && computerMove == SCISSORS) {
+            return true;
+        } else if (playerMove == LIZARD && computerMove == LIZARD) {
+            return true;
+        } else if (playerMove == SPOCK && computerMove == SPOCK) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public void play() {
+        Scanner input = new Scanner(System.in);
+        Dialogs.startMenu();
+        playerName = Dialogs.setPlayerName();
+        pointsToWin = Dialogs.setRoundsToWin();
+        while(!end) {
+            playerMove = Dialogs.setPlayerMove();
+            if (playerMove == NEW_GAME) {
+                System.out.println("Starting new game!");
+                playerScore = 0;
+                computerScore = 0;
+                play();
+            }
+            if (playerMove == END_GAME) {
+                System.out.println("Are you sure? Y - yes, N - no");
+                String choice = input.nextLine().toUpperCase();
+                if (choice.equals("Y")) {
+                    System.out.println("Ending game. Application will be closed!");
+                    end = true;
+                } else if (choice.equals("N")) {
+                    continue;
+                } else {
+                    System.out.println("Choose Y/N");
+                    continue;
+                }
+            }
+            if (playerMove == ROCK || playerMove == PAPER || playerMove == SCISSORS || playerMove == LIZARD || playerMove == SPOCK) {
+                computerMove = setComputerMove();
+                if (playerWin(playerMove, computerMove)) {
+                    System.out.println("Player choose: " + playerMove + " VS " + computerMove + "   " + playerName + " won!");
+                    playerScore++;
+                    System.out.println("Current score: " + playerName + " " + playerScore + ":" + computerScore + " COMPUTER");
+                } else if (computerWin(playerMove, computerMove)) {
+                    System.out.println("Player choose: " + playerMove + " VS " + computerMove + "   " + "COMPUTER won!");
+                    computerScore++;
+                    System.out.println("Current score: " + playerName + " " + playerScore + ":" + computerScore + " COMPUTER");
+                } else if (draw(playerMove, computerMove)) {
```

Comment:
czy potrzebny nam draw? Jesli nie wygral komputer ani gracz, to chyba wynika z tego, ze jest remis?